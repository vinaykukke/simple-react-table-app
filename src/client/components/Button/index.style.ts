import styled from 'styled-components';

export const Button = styled.button`
  height: 30px;
  background: #3a8de0;
  border-radius: 8px;
  margin-left: 1em;
  border: none;
  color: white;
  cursor: pointer;
`;
