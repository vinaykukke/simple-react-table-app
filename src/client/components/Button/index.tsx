import React from 'react';
import * as S from './index.style';

const Button = (props: any) => {
  return <S.Button {...props}>{props.title}</S.Button>
};

export default Button;
