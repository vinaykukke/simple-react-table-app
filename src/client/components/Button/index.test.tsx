import React from 'react';
import renderer from 'react-test-renderer';

import Button from './index';

describe('Button Component', () => {
  it('should render <Button />', () => {
    const tree = renderer.create(<Button title='test'/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
