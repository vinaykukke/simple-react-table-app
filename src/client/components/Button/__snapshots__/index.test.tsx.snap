// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component should render <Button /> 1`] = `
<button
  className="sc-bdVaJa cqgoro"
  title="test"
>
  test
</button>
`;
